GOVERNANCE.md

```diff
@@ -0,0 +1,104 @@
+  goal.
+
+## Your organization is currently a maintainer
+
+* First decide whether your organization really needs more people with maintainer access. Valid
+  reasons are "blast radius", a large organization that is working on multiple unrelated projects,
+  etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard"
+  maintainer access.
+* "Standard" maintainer access can be upgraded to "senior" maintainer access after another 1-2
+  months of work and another conference of the existing senior committers.
+
+## Maintainer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+* During GitHub issue triage, apply all applicable [labels](https://github.com/tikv/tikv/labels)
+  to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply
+  is somewhat subjective so just use your best judgment. A few of the most important labels that are
+  not self explanatory are:
+  * **beginner**: Mark any issue that can reasonably be accomplished by a new contributor with
+    this label.
+  * **help wanted**: Unless it is immediately obvious that someone is going to work on an issue (and
+    if so assign it), mark it help wanted.
+  * **question**: If it's unclear if an issue is immediately actionable, mark it with the
+    question label. Questions are easy to search for and close out at a later time. Questions
+    can be promoted to other issue types once it's clear they are actionable (at which point the
+    question label should be removed).
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.
+* Participate when called upon in the security release process. Note
+  that although this should be a rare occurrence, if a serious vulnerability is found, the process
+  may take up to several full days of work to implement. This reality should be taken into account
+  when discussing time commitment obligations with employers.
+* In general continue to be willing to spend at least 25% of ones time working on TiKV (~1.25
+  business days per week).
+* We currently maintain an "on-call" rotation within the maintainers. Each on-call is 1 week.
+  Although all maintainers are welcome to perform all of the above tasks, it is the on-call
+  maintainer's responsibility to triage incoming issues/questions and marshal ongoing work
+  forward. To reiterate, it is *not* the responsibility of the on-call maintainer to answer all
+  questions and do all reviews, but it is their responsibility to make sure that everything is
+  being actively covered by someone.
+
+## Cutting a release
+
+* We do releases approximately every 6 weeks.

```

Comment:
Move this out of the list. And add:
"We do releases approximately every 6 weeks. Here is how a regular release goes out:"